server/protocols/jmap/src/test/java/org/apache/james/jmap/model/CreationMessageTest.java

```diff
@@ -55,6 +51,11 @@ public void validateShouldReturnErrorWhenFromIsMissing() {
 
     @Test
     public void validateShouldReturnErrorWhenFromIsInvalid() {
+        testedBuilder = CreationMessage.builder()
+                .mailboxIds(ImmutableList.of("ba9-0f-dead-beef"))
+                .headers(ImmutableMap.of())
```

Comment:
I think you can keep these 3 lines in the setUp, it will reduce the size of your tests.